allure-rest-assured/src/main/java/io/qameta/allure/restassured/AllureLoggerFilter.java

```diff
@@ -0,0 +1,68 @@
+package io.qameta.allure.restassured;
+
+import io.qameta.allure.curl.CurlBuilder;
+import io.qameta.allure.curl.FreemarkerUtils;
+import io.restassured.filter.Filter;
+import io.restassured.filter.FilterContext;
+import io.restassured.internal.NameAndValue;
+import io.restassured.response.Response;
+import io.restassured.specification.FilterableRequestSpecification;
+import io.restassured.specification.FilterableResponseSpecification;
+import ru.yandex.qatools.allure.Allure;
+import ru.yandex.qatools.allure.events.MakeAttachmentEvent;
+import ru.yandex.qatools.allure.events.StepFinishedEvent;
+import ru.yandex.qatools.allure.events.StepStartedEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AllureLoggerFilter implements Filter {
```

Comment:
It would be better to implemet OrderedFilter interface instead of just Filter and return `Integer.MAX_VALUE` for `getOrder()` because it guarantees that logger will be the last one in filters list (or at least one of the filters with highest order) and show actual request data.